File: High-Access Employees.java
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

class Solution {
    public List<String> findHighAccessEmployees(List<List<String>> access_times) {
        Map<String, List<Integer>> mp = new HashMap<>();
        for (List<String> it : access_times) {
            int time = Integer.parseInt(it.get(1));
            mp.computeIfAbsent(it.get(0), k -> new ArrayList<>()).add(time);
        }
        for (List<Integer> times : mp.values()) {
            times.sort(null);
        }
        List<String> ans = new ArrayList<>();
        for (Map.Entry<String, List<Integer>> entry : mp.entrySet()) {
            List<Integer> times = entry.getValue();
            for (int i = 2; i < times.size(); i++) {
                if (times.get(i) - times.get(i - 2) < 100) {
                    ans.add(entry.getKey());
                    break;
                }
            }
        }
        return ans;
    }
}

File: High-Access Employees.py
class Solution:
    def findHighAccessEmployees(self, access_times):
        mp = {}
        for it in access_times:
            time = int(it[1])
            if it[0] not in mp:
                mp[it[0]] = []
            mp[it[0]].append(time)
        for key in mp:
            mp[key].sort()
        ans = []
        for key in mp:
            times = mp[key]
            for i in range(2, len(times)):
                if times[i] - times[i - 2] < 100:
                    ans.append(key)
                    break
        return ans


